### DISCUSSION TITLE
Missing pieces

### DISCUSSION THREAD

**Discussion by Nathan458** [leozide/leocad#909] [Q&A]
I've noticed that there are some missing pieces. One such piece is piece "27262", which I cannot complete my build without it.
![250x250p](https://github.com/leozide/leocad/assets/147889707/01e78b1c-f2c1-47ae-91a9-c3b50c2eefbb)

**Comment by cuddlyogre** [comment]
That looks to be an unofficial part which you'll need to download separately until it gets added to the official library.

https://library.ldraw.org/tracker/24320

**Reply by Nathan458** [reply]
And also, how do you add it to LeoCAD, I got my LeoCAD by using the command: "sudo apt-get install leocad" because I use Ubuntu Linux and when I try to find the execution file in my text editor for it is shows me this "Exec=leocad %f".

**Reply by cuddlyogre** [reply]
You'll need to change the library LeoCAD uses.

In LeoCAD, go to View > Preferences > General > Parts Library

![image](https://github.com/leozide/leocad/assets/3506795/0ae66255-6d7e-4db9-aa32-f3120563d1d9)

If you choose icon 1, you can set the path to the folder that your LDraw library is in, instead of using the zip. Then download the part into **LDRAW_ROOT/ldraw/Unofficial/parts/**. You may need to create that folder. If you don't have an LDraw folder, you can extract the zip and use that.

This unofficial part likely has other unofficial dependencies, meaning it won't load, so it would be faster for you to download the whole unofficial library and extract that into **LDRAW_ROOT/ldraw/Unofficial/**

**Reply by Nathan458** [reply]
Thank you so much!

**Comment by nathaneltitane** [comment]
@leozide documentation needs a drastic update - how do we go about making this clear once and for all?

**Comment by SLENDERFOL** [comment]
I noticed that the kit lacks the part number "6452845", without this part I cannot continue to assemble the Aston Martin Valkirya, I stopped at step 60 of the assembly
